decidim-sortitions/app/views/decidim/sortitions/sortitions/show.html.erb

```diff
@@ -70,9 +70,7 @@ edit_link(
           </span>
         <% end %>
       </div>
-      <div class="callout alert">
-        <%= decidim_sanitize_editor translated_attribute sortition.cancel_reason %>
-      </div>
+      <%= cell("decidim/announcement", decidim_sanitize_editor(translated_attribute(sortition.cancel_reason)), callout_class: "alert") %>
```

Comment:
Yes, that can't be read for sure. I asked to @carolromero and we agreed to just remove the truncate parameter, as it was only used in two places and we preferred to just remove it instead of fixing it. 
(The other place was in the budget component announcement, but why only for this component and not others? 🤷🏽) 
It was changed in 887f1f36bde89d1bf5bff6a17b4bb3012a0f1aef

Now it looks like this: 
![Screenshot of the sortitions explanation with the change](https://github.com/decidim/decidim/assets/717367/7780a460-ee13-4051-a03f-2c706c13fb14)

Reply:
great! Confirmed.